lib/web3-utils.js

```diff
@@ -0,0 +1,20 @@
+const RLP = require('rlp')
+
+// functions for counterfactual addresses
+async function buildNonceForAddress(_address, _index) {
```

Comment:
Nit: are we using the `_` as a prefix for these JS helpers usually?

Reply:
Not really, my bad; forgot to change these when I moved them from a Discord conversation!

Reply:
Done in https://github.com/aragon/court-precedence-campaign/pull/3/commits/3f4f8a23c7e71b4eeb561194ca4d221f98428425